components/NotificationBar.js

```diff
@@ -1,12 +1,63 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import styled from 'styled-components';
 
 import AcceptRejectButtons from './host-dashboard/AcceptRejectButtons';
+import Container from './Container';
 import { Flex } from './Grid';
+import { H1, P } from './Text';
 
 const logo = '/static/images/opencollective-icon.svg';
 
+const NotificationBarContainer = styled.div`
+    .oc-message .logo {
```

Comment:
We should ideally get rid of these CSS classes, use dedicated styled components where we can.

Reply:
@Betree : I've refactored to get rid of the CSS classes as much as possible. Take a look and let me know if there's anything else to be done. 😄